src/box/lua/config/utils/aboard.lua

```diff
@@ -14,6 +14,11 @@ local function aboard_set(self, key, alert)
     self._alerts[key] = alert
 end
 
+local function aboard_add(self, alert)
```

Comment:
Nit: it doesn't look good to add the new function, which basically does the same as `aboard_set`. Maybe we can pass key to `aboard_set` as a second option? If it's not passed, then we'll use `_next_key` automatically. Up to you, not a principal issue for me

Reply:
I agree, the methods are doing very similar things and could be merged into one method.

I've placed it into the `opts` table, so it'll be more obvious what is the string at the caller side.

```lua
aboard:set({type = 'warn', message = '<...>'}, {key = '<...>'})
```